app/models/ZkKafka.scala

```diff
@@ -148,4 +160,14 @@ object ZkKafka {
 
     (new Totals(total, kafkaTotal, spoutTotal), deltas)
   }
+
+  // Attempt to parse a byte array as json, and catch and log any error rather than
+  // bubbling it up.
+  def tryParse(json: Array[Byte]): Option[JsValue] = try {
+    Some(Json.parse(json))
+  } catch {
+    case e: Exception =>
+      Logger.error(s"failed parsing json document:\n${json.take(100)}", e)
```

Comment:
From my testing Scala still does not support %n in interpolated strings in 2.10 or 2.11, despite a language bug having been reported (and supposedly fixed) in the 2.10.2 release.  The argument can be changed, though, to explicitly call ".format()" which does properly handle it.